js/components/graph/Node.js

```diff
@@ -200,7 +214,7 @@ export default class Node extends React.Component {
     })
     this.props.inEdges.forEach(function (edge) {
       var currentEdge = svg.edges.current[edge]
-      if (currentEdge.state.status === "missing") {
+      if (currentEdge.props.status === "missing") {
```

Comment:
As a comment, this line couldn't have been working properly before because you had named the prop `edgeStatus` instead of status. This is a good example of a case where tests should catch an error.